doc.go

```diff
@@ -131,7 +131,9 @@ The request has a valid URL but the method (GET,PUT,POST,...) is not allowed.
 
 	406: Not Acceptable
 
-The request does not have or has an unknown Accept Header set for this operation.
+The request does not have or has an unknown Accept Header set for this operation. Note that a 406 might also occur
```

Comment:
please clarify the word "might" in this context.  IMHO it either works or not. Could you provide an example testcase that demonstrates the problem and solution?